src/aap_eda/analytics/analytics_collectors.py

```diff
@@ -40,6 +54,76 @@ def config(**kwargs) -> dict:
     }
 
 
+@register(
+    "activations_stats",
+    "1.0",
+    format="csv",
+    description="Stats for activations",
+)
+def activation_stats(
+    since: datetime, full_path: str, until: datetime, **kwargs
+):
+    has_ended_at_states = [
+        ActivationStatus.FAILED.value,
+        ActivationStatus.STOPPED.value,
+        ActivationStatus.COMPLETED.value,
+        ActivationStatus.UNRESPONSIVE.value,
+        ActivationStatus.ERROR.value,
+        ActivationStatus.WORKERS_OFFLINE.value,
+    ]
+    counts_query = (
+        models.RulebookProcess.objects.filter(activation_id=OuterRef("pk"))
+        .values("activation_id")
```

Comment:
Minor: I think this line is not necessary since you already are counting the id later